newsapi_headlines-to-email/main.bal

```diff
@@ -0,0 +1,30 @@
+import ballerina/log;
+import ballerinax/newsapi;
+import wso2/choreo.sendemail;
+
+//News API configuration
+configurable newsapi:ApiKeysConfig apiKeyConfig = ?;
+configurable string emailAddress = ?;
+
+public function main() returns error? {
+    string mailBody = "";
+    newsapi:Client newsClient = check new (apiKeyConfig, {}, "https://newsapi.org/v2");
+    newsapi:WSNewsTopHeadlineResponse topHeadlines = check newsClient->listTopHeadlines(sources="bbc-news", page=1);
+    log:printInfo(topHeadlines.toString());
+    newsapi:WSNewsArticle[]? articles = topHeadlines?.articles;
+    if articles is newsapi:WSNewsArticle[] && articles.length() != 0 {
+        mailBody = "BBC top news are,\n";
+        foreach var article in articles {
```

Comment:
Avoid using var.

Reply:
Updated in [5131705](https://github.com/ballerina-guides/integration-samples/pull/35/commits/51317055927a041c645786b20bcb6af3864f1081)